src/openforms/prefill/contrib/eidas/tests/test_config_checks.py

```diff
@@ -0,0 +1,114 @@
+from django.test import TestCase
+from django.utils.translation import gettext_lazy as _
+
+from mozilla_django_oidc_db.tests.factories import OIDCProviderFactory
+
+from openforms.contrib.auth_oidc.tests.factories import OFOIDCClientFactory
+from openforms.plugins.exceptions import InvalidPluginConfiguration
+from openforms.prefill.contrib.eidas.plugin import EIDASPrefill
+from openforms.utils.tests.oidc import OIDCMixin
+
+
+class EIDASPrefillConfigCheckTests(OIDCMixin, TestCase):
+    def setUp(self):
```

Comment:
don't forget to call `super().setUp()`, *especially* because `OIDCMixin` is used which sets up some mocks

I've also merged #5550 which moved the import path for `OIDCMixin`, so please rebase on master and update :)